src/views/Register.vue

```diff
@@ -64,5 +66,25 @@ export default {
       password: null,
     };
   },
+  methods: {
+    register() {
+      if (!(this.nome && this.email && this.password)) {
+        console.log('Preencha todas as informações.');
+        return;
+      }
+
+      try {
+        axios.post('http://localhost:3000/users/', { name: this.nome, email: this.email, password: this.password })
```

Comment:
para manter o padrão do resto do código a requisição poderia ser feita com: this.$http.post('/user', ~aqui passar as info necessarias~)

Reply:
Feito!